src/rpc/net.cpp

```diff
@@ -290,8 +290,7 @@ static RPCHelpMan addnode()
     if (!request.params[1].isNull())
         strCommand = request.params[1].get_str();
     if (strCommand != "onetry" && strCommand != "add" && strCommand != "remove") {
-        throw std::runtime_error(
-            self.ToString());
+        throw std::runtime_error(self.ToString());
```

Comment:
Due to the amount of files changed and the context of this PR, perhaps we could leave this out(?) and follow it up in a different change.